_posts/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2.markdown

```diff
@@ -1,55 +1,80 @@
+custom AMI.
 
-Our AMI setup steps are captured entirely in a [GitHub gist](https://gist.github.com/3085368), but because our build requirements are specific to our applications and frameworks, most organizations will need to modify these scripts to their own use cases. Given that caveat, here's how we went from base Ubuntu AMI to custom build slave AMI:
+Our AMI setup steps are captured entirely in a [GitHub gist](https://gist.github.com/3085368), but because our
+build requirements are specific to our applications and frameworks, most organizations will need to modify these
+scripts to their own use cases. Given that caveat, here's how we went from base Ubuntu AMI to custom build agent
+AMI:
 
-1. We [launched](https://console.aws.amazon.com/ec2/home?region=us-east-1#launchAmi=ami-4dad7424) an Ubuntu 11.10 AMI `4dad7424` via the AWS console.
-2. Once the instance was launched, we logged in with the SSH key we generated during setup.
-3. We ran the following commands to configure the instance:
+1.  We [launched](https://console.aws.amazon.com/ec2/home?region=us-east-1#launchAmi=ami-4dad7424) an Ubuntu 11.10
+    AMI `4dad7424` via the AWS console.
+2.  Once the instance was launched, we logged in with the SSH key we generated during setup.
+3.  We ran the following commands to configure the instance:
 
         curl -L https://raw.github.com/gist/3085368/_base-setup.sh | sudo bash -s
         sudo su -l jenkins
         curl -L https://raw.github.com/gist/3085368/_jenkins-user-setup.sh | bash -s
 
-4. From the "Instances" tab of the AWS Console, we chose the now-configured instance, and from the "Instance Actions" dropdown, selected "Stop", followed by "Create Image (EBS AMI)".
+4.  From the "Instances" tab of the AWS Console, we chose the now-configured instance, and from the "Instance
+    Actions" dropdown, selected "Stop", followed by "Create Image (EBS AMI)".
 
-Next we installed the Amazon EC2 Plugin on our Jenkins master, and entered the following configuration arguments for the plugin. (Replace the AMI ID with your own, the result of Step 4 above.)
+Next we installed the Amazon EC2 Plugin on our Jenkins controller, and entered the following configuration
+arguments for the plugin. (Replace the AMI ID with your own, the result of Step 4 above.)
 
-![/images/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2/ec2-plugin-config.png](Jenkins EC2 Plugin configuration)
+![/images/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2/ec2-plugin-config.png](Jenkins EC2 Plugin
+configuration)
 
-New build slaves began spawning immediately in response to job demand! Our new "Computers" page on Jenkins looks like this:
+New build agents began spawning immediately in response to job demand! Our new "Computers" page on Jenkins looks
+like this:
 
 ![/images/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2/computer-list.png](Jenkins computer list)
```

Comment:
**question (non-blocking):** Should we also update this filename?

Reply:
Yes, but I think that should go with the PR we put up for the redirect!